### PR TITLE
Tutorial deep

### PR DESCRIPTION
**Pull request by riccardodv**
We created a tutorial/quick start for deep agents (A2C).

### CODE REVIEW

File: docs/basics/DeepRLTutorial/TutorialDeepRL.rst [hunk 1]
```diff
@@ -0,0 +1,605 @@
...
+    [INFO] [eval]... simulation 49/50
+    [INFO] [eval]... simulation 50/50
+
+
+
+.. image:: output_6_3.png
+
+
+Let’s try to change the neural networks’ architectures and see if we can
+beat our previous result. This time we we use a smaller learning rate
```
Comment (TimotheeMathieu) [inline docs/basics/DeepRLTutorial/TutorialDeepRL.rst:332]: typo we we

**Review by TimotheeMathieu — COMMENTED** [review]
Looks good !
A few typo, otherwise nice.

File: docs/basics/DeepRLTutorial/TutorialDeepRL.rst [hunk 2]
```diff
@@ -0,0 +1,605 @@
+Deep Reinforecement Learning in rlberry
+=======================================
+
```
Comment (TimotheeMathieu) [inline docs/basics/DeepRLTutorial/TutorialDeepRL.rst:3]: Remark : if you want to have the names of the authors in a comment, in rst this is done like that:
```
..
  Authors: Riccardo, Alena, Hector.
```

File: docs/basics/DeepRLTutorial/TutorialDeepRL.rst [hunk 3]
```diff
@@ -0,0 +1,605 @@
...
+In this tutorial we will focus on Deep Reinforcement Learning with an
```
Comment (mmcenta) [inline docs/basics/DeepRLTutorial/TutorialDeepRL.rst:4]: I think the name is in the wrong order, it should be: "In this tutorial, we will focus on Deep Reinforcement Learning with the **Advantage Actor-Critic** algorithm."

File: docs/basics/DeepRLTutorial/TutorialDeepRL.rst [hunk 4]
```diff
@@ -0,0 +1,605 @@
...
+    from rlberry.envs import gym_make
+    from rlberry.manager import plot_writer_data, AgentManager, evaluate_agents
+    from rlberry.agents.torch import A2CAgent
+    from rlberry.agents.torch.utils.training import model_factory_from_env
+
+
+Reminder of the RL setting
+--------------------------
+
+As always we will consider a MDP
```
Comment (mmcenta) [inline docs/basics/DeepRLTutorial/TutorialDeepRL.rst:21]: Maybe remove the "as always".

File: docs/basics/DeepRLTutorial/TutorialDeepRL.rst [hunk 5]
```diff
@@ -0,0 +1,605 @@
...
+.. math::  J(\pi) = \mathbb{E}_{\tau \sim \pi}\big[R(\tau) \big]
+
+Gym Environment
+---------------
+
+In this tutorial we are going to use the `OpenAI’s Gym
+library <https://gym.openai.com/envs/>`__. This library provides a large
+number of environments to test RL algorithm.
+
+We will focus on the **CartPole-v1** environment, anyway, it might be
```
Comment (mmcenta) [inline docs/basics/DeepRLTutorial/TutorialDeepRL.rst:50]: Suggestion: "We will focus only on the **CartPole-v1** environment, although we recommend experimenting with other environments such as **Acrobot-v1** and **MountainCar-v0**."

File: docs/basics/DeepRLTutorial/TutorialDeepRL.rst [hunk 6]
```diff
@@ -0,0 +1,605 @@
...
+Actor-Critic algorithms and A2C
+-------------------------------
+
+**Actor-Critic algorithms** methods consist of two models, which may
+optionally share parameters: - Critic updates the value function
+parameters w and depending on the algorithm it could be action-value
+:math:`Q_{\varphi}(s,a )` or state-value :math:`V_{\varphi}(s)`. - Actor
+updates the policy parameters :math:`\theta` for
+:math:`\pi_{\theta}(a \mid s)`, in the direction suggested by the
+critic.
```
Comment (mmcenta) [inline docs/basics/DeepRLTutorial/TutorialDeepRL.rst:74]: Should this be a list? Because it's displayed as a paragraph.

File: docs/basics/DeepRLTutorial/TutorialDeepRL.rst [hunk 7]
```diff
@@ -0,0 +1,605 @@
...
+
+-  a multilayer perceptron with two hidden layers of 64 units for the
+   **Actor**, the input layer has the dimension of the state space while
+   the output layer has the dimension of the action space. The
+   activations are RELU functions and we have a softmax in the last
+   layer.
+-  a multilayer perceptron with two hidden layers of 64 units for the
+   **Critic**, the input layer has the dimension of the state space
+   while the output has dimension 1. The activations are RELU functions
+   apart from the last layer that has a linear activation.
```
Comment (mmcenta) [inline docs/basics/DeepRLTutorial/TutorialDeepRL.rst:142]: Minor detail, but I think people are referring to MLPs more as dense neural networks nowadays.

**Review by mmcenta — CHANGES_REQUESTED** [review]
I made some minor suggestions, let me know what you think.

**Review by mmcenta — APPROVED** [review]
LGTM!
